docs/build-with-powerloom/use-cases/existing-implementations/uniswap-dashboard/data-points.md

```diff
@@ -4,15 +4,15 @@ sidebar_position: 1
 
 # Datapoints
 
-![UniswapV2 Dashboard Powerloom](/images/uniswapv2-dashboard-powerloom.png)
+![UniswapV2 Dashboard Powerloom](/images/uniswapv2-dashboard-powerloom-2.png)
 
-In Pooler, data points are specific, quantifiable elements derived from Uniswap V2 trading activities. 
+In Pooler, data points are specific, quantifiable elements derived from Uniswap V2 and V3 trading activities. 
 Each snapshot inherits `Snapshot Base` which looks something like this
 ```python reference
 https://github.com/PowerLoom/snapshotter-computes/blob/6fb98b1bbc22be8b5aba8bdc860004d35786f4df/utils/models/message_models.py#L9-L17
 ```
 
-The following type of Base Snapshots are generated by Pooler for Uniswap V2 Dashboard
+The following type of Base Snapshots are generated by Pooler for the Uniswap V2 and V3 Dashboards
 
 **1. Pair Reserves Snapshot:** Liquidity reserves of each pair per Epoch. The reserves snapshot looks something like this
```

Comment:
these require the right H2, H3 etc formatting to show up on the side pan.

Also, don't see the Q notation capture of the snapshots for pair reserves described here which is very specific to Uniswap V3 and worth a mention.

Should also include a link to the Uniswap Labs' primer article on Uniswap V3 math.

Reply:
The proper heading formatting has been added here:

https://github.com/PowerLoom/docs/commit/a2e22ee2b155c75dd2b793f53a8acc4e2bdce646

The Pair Reserves Snapshot differences between V2 and V3 have been expanded upon here:

https://github.com/PowerLoom/docs/commit/1936fe73556aadf847629039d7bc76748172f609